### ISSUE TITLE
[BUG] [Java Service]sourceFormat value can't accept lower-case in string-based GET/POST APIs.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1557]
**Describe the bug**
commit:2a16c214690243b83109a776751ed50773689bf7
sourceFormat value can't accept lower-case in string-based GET/POST APIs.
The issue is reproducible in all v1/v2 string-based GET/POST APIs.

**To Reproduce**
1. Go to 'GET
[/i18n/api/v2/translation/products/{productName}/versions/{version}/locales/{locale}/components/{component}/keys/{key}]'
2. Request a key with "md" sourceFormat as below:
http://localhost:8091/i18n/api/v2/translation/products/VMwareVIP2018/versions/1.1.0/locales/en/components/markdown/keys/md.key-1?collectSource=false&pseudo=true&sourceFormat=md
3. See error
```
{
  "response": {
    "code": 400,
    "message": "Incorrect sourceformat(only allows empty, MD, HTML, SVG)",
    "serverTime": ""
  },
  "signature": "",
  "data": null
}
```

**Expected behavior**
sourceFormat value should be case insensitive in string-based GET/POST APIs.
Can accept "String", "md", "Md", "mD" such value.

**Comment by tigershi** [comment]
The source format value only allows empty, MD, HTML, SVG.

**Comment by lyiyu66** [comment]
The issue is fixed with commit 653b579d60fd4592493784e7c3f023d6b8372d6b
